style.css

```diff
@@ -0,0 +1,205 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+.header {
+  position: absolute;
+  width: 375px;
```

Comment:
- [ ] One of the requirements for this project is to have a responsive design up until the desktop screen that starts from 768px breakpoint. as you have given a fixed width of 375px in your header and headline section, this makes your design unresponsive. kindly make sure that your design is responsive.
the requirement is:  
- mobile version should be responsive until the desktop screen size.
your design
![qwer](https://user-images.githubusercontent.com/96683843/212157596-d5a9e521-a6d2-417d-8967-3c7250d5a046.jpg)